### PR TITLE
Make Drawables

### PR DESCRIPTION
**Pull request by biyeun**
A proof of concept for png drawables management. (Not yet doing anything for production code, but more of an FYI for peeps to see where I'm heading with this)

### CODE REVIEW

**Comment by biyeun** [comment]
fyi @orangejenny

**Comment by biyeun** [comment]
@wpride is there any more information you need from me on this?

**Comment by wpride** [comment]
naw thanks. looks great

**Comment by ctsims** [comment]
Oh, biyeun, can you actually flag this PR as "Ready for Review"? We've been
trying to use the flags to signal when the PR is actually ready to be
pulled (v. ready to be reviewed)

-Clayton

File: images/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Test: `python make_drawables.py --test`
+
+## How to create resources
+
+Put the src-images in `./src-images`. They must be `.png` at this time.
+(`.svg` and `.9.png` support coming soon)
+
+Put the spec as `.yaml` file for each source image, indicating the sizes
+for each destination folder.
```
Comment (orangejenny) [inline images/README.md]: sample YAML would be nice

File: images/dpi_manager.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        src_img = Image.frombytes("RGBA", src_img.size, premult.tobytes())
+
+        # save original image to drawables
+        default_dir = os.path.join(self.target_folder, DRAWABLE)
+        ensure_dir(default_dir)
+        default_path = os.path.join(default_dir, self.spec.filename)
+        src_img.save(default_path)
+        print "save to", default_path
+
+        src_width, src_height = src_img.size
```
Comment (orangejenny) [inline images/dpi_manager.py]: square images only?

File: images/make_drawables.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+from utils import (
+    ensure_dir,
+)
+
+
+def main(args):
+    test = False
+
+    if '--help' in args:
+        print("--dry-run: dry run mode, outputs to ./test folder")
```
Comment (orangejenny) [inline images/make_drawables.py]: shouldn't this be --test, and shouldn't the script exit after printing help info?

**Comment by orangejenny** [comment]
looks reasonable, couple of small comments

File: images/make_drawables.py (hunk 3 above)
Comment (biyeun) [inline images/make_drawables.py]: yep...correct

File: images/README.md (hunk 1 above)
Comment (biyeun) [inline images/README.md]: there is one? `images/spec/icon.yaml` Do you mean in the readme?

File: images/dpi_manager.py (hunk 2 above)
Comment (biyeun) [inline images/dpi_manager.py]: how is that square?

File: images/README.md (hunk 1 above)
Comment (orangejenny) [inline images/README.md]: yeah, I mean in the readme

File: images/dpi_manager.py (hunk 2 above)
Comment (orangejenny) [inline images/dpi_manager.py]: Aren't you setting both width and height to size?

Comment (biyeun) [inline images/dpi_manager.py]: so, `src_img.size` outputs a 2 element tuple, and this is how you two different variables to a tuple in python. It's pretty standard.

Comment (orangejenny) [inline images/dpi_manager.py]: ah, didn't realize size returns a tuple, got it.

**Pull request merged**